src/Malawi_cor_labels_GEE script.js

```diff
@@ -0,0 +1,342 @@
+    var lat = point.get(1);
+    return feature.set({
+      longitude: long,
+      latitude: lat,
+    });
+  });
+  
+  //Generate a URL for the points and export as CSV
+  //button.setLabel('Saved!');
+  downloadUrl.setValue('');
+  button.setDisabled(true);
+  sampled.getDownloadURL({
+    filename: 'Malawi_new points',
+    callback: function(url) {
+      downloadUrl.setValue(url);
+      downloadUrl.setUrl(url);
+      button.setDisabled(false);
+    }
+  });
+  
+});
+
+// Create the split panels
+var splitPanel = ui.SplitPanel({
+  firstPanel: leftMap,
+  secondPanel: rightMap,
+  orientation: 'horizontal',
+  wipe: false,
+  style: {stretch: 'both'}
+});
+
+
+// Add captions
+var infoPanel1 = ui.Panel({
+  style: {
+    position: 'bottom-left',
+    padding: '8px 15px',
+    stretch: "both",
+  }
+});
+
+var infoPanel2 = ui.Panel({
+  style: {
+    position: 'bottom-right',
+    padding: '8px 15px',
+    stretch: "both",
+  }
+});
+
+var info1 =ui.Label({
+  value: "Malawi cropland map",
+  style: {
+    fontSize: '12px',
+    margin: '0 0 3px 0',
+    padding: '0'
+  }
+});
+
+var info2 =ui.Label({
+  value: "Malawi Sentinel 2 Image",
+  style: {
+    fontSize: '12px',
+    margin: '0 0 3px 0',
+    padding: '0'
+  }
+});
+
+var title = ui.Panel({
+  widgets: [
+    ui.Label("", {
+      stretch: 'horizontal',
+    }),
+    ui.Label("Corrective Labeling App - Malawi", {
+      fontSize: '24px',
+      fontWeight: 'bold',
+      //fontWeight: '120',
+    }),
+    ui.Label("", {
+      stretch: 'horizontal',
+    }),
+  ],
+  layout: ui.Panel.Layout.flow('horizontal'),
+});
+
+//Adding a Legend
+// set position of panel
+var legend = ui.Panel({
+  style: {
+    position: 'bottom-left',
+    padding: '8px 10px',
+    stretch: 'horizontal'
+  }
+});
+ 
+// Create legend title
+var legendTitle = ui.Label({
+  value: 'Legend',
+  style: {
+    fontWeight: 'bold',
+    position: 'bottom-left',
+    fontSize: '15px',
+    margin: '0 0 4px 0',
+    padding: '0'
+    }
+});
+
+// Creates and styles 1 row of the legend.
+var makeRow = function(color, name) {
+ 
+      // Create the label that is actually the colored box.
+      var colorBox = ui.Label({
+        style: {
+          backgroundColor: '#' + color,
+          // Use padding to give the box height and width.
+          padding: '8px',
+          margin: '0 0 4px 0'
+        }
+      });
+ 
+      // Create the label filled with the description text.
+      var description = ui.Label({
+        value: name,
+        style: {margin: '0 0 4px 6px'}
+      });
+ 
+      // return the panel
+      return ui.Panel({
+        widgets: [colorBox, description],
+        layout: ui.Panel.Layout.Flow('horizontal')
+      });
+};
+legend.add(legendTitle); 
+//  Palette with the colors
+var palette =['197228', 'FFFDCE'];
+ 
+// name of the legend
+var names = ['Crop','Non Crop',];
+ 
+//Add color and and names
+for (var i = 0; i < 2; i++) {
+  legend.add(makeRow(palette[i], names[i]));
+  }  
```

Comment:
I think this would be easier to read if it was condensed. Also if there's only two examples it takes fewer lines to just write out the code
```javascript
var makeRow = function(color, name) {
  var colorBoxStyle = {backgroundColor: '#' + color, padding: '8px', margin: '0 0 4px 0'}
  var colorBox = ui.Label({style: colorBoxStyle});
  var description = ui.Label({value: name, style: {margin: '0 0 4px 6px'}});
  return ui.Panel({
    widgets: [colorBox, description],
    layout: ui.Panel.Layout.Flow('horizontal')
  });
};
legend.add(makeRow('197228', 'Crop'))
legend.add(makeRow('FFFDCE', 'Non Crop'))
```

Reply:
Also the tabbing in the original function is off

Reply:
Good catch, thanks